ionic-template/Routes.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Route, useRouteMatch } from 'react-router-dom'
```

Comment:
We have `react-router` as a peer dependency, not `react-router-dom`. Do we need to modify that?

Reply:
@mwislek bump